src/components/layout/Layout.tsx

```diff
@@ -31,16 +31,16 @@ export const Layout = ({
   searchInput,
   sidebar,
 }: LayoutProps) => (
-  <main className={[className, layoutStyles.aminoLayout].join(' ')}>
+  <main
+    className={[className, layoutStyles.aminoLayout].join(' ')}
+    style={{
+      '--height': headerContent ? `calc(100vh - ${theme.appbarHeight})` : '',
```

Comment:
`100vh` is better be the fallback here instead of empty string, because empty string is a valid css variable, and it will not fallback to `100vh` as you have in `var(--height, 100vh)`, the compiled css would just be *~~height: ''~~*. And then in the case you just don't want to set the value (like you want to keep , use `initial` instead of empty string to just use the default property value from the browser or `inherit` to use the style from the parent.

Reply:
Ah I see. Getting my JS and CSS mixed up